.github/workflows/update-leaderboard.yml

```diff
@@ -0,0 +1,80 @@
+name: Update speedrun leaderboard
```

Comment:
Can we make one script that does all of these bookkeeping commands, so that this is basically one script?  I think for now, it's not important to have full automation - it just has to be easy for us to run a command.  Ideally, we would want to have the option of submitting a PR and double checking the results before we commit to the public website, just in case something breaks.